client/src/components/ProjectSearchFilter.js

```diff
@@ -89,6 +91,26 @@ export const ProjectSearchFilter = ({
 
   return (
     <Box overflow="auto">
+      {filterOptions.models.map((f, i, arr) => (
+        <Box
+          key={f}
+          border={!isLastIndex(i, arr) ? false : { side: 'bottom' }}
+          pad={{
+            top: `${i === 0 ? 'medium' : 'none'}`,
+            bottom: `${!isLastIndex(i, arr) ? 'none' : 'medium'}`
+          }}
```

Comment:
You could remove this because there shouldn't be line between these line items. I would just use the `gap` prop on the parent `Box` for spacing. Also, you can just put the border on the bottom or the parent `Box`.

Reply:
Done 🎉